app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Api::V1::DoctorsController < ApiController
+  # before_action :authenticate_user!
+
+  # GET /doctors
+  def index
+    @doctors = Doctor.all
+    if @doctors
+      render json: { success: true, message: 'Loaded all doctors', data: { doctors: @doctors } }, status: :ok
```

Comment:
It may be better to have a method handle messages where you pass the type of message and contest etc. There is a lot of repeating with rendering json in your controllers. It'd look cleaner using a method to render your messages with message content, type of message (error etc.)